docs/pages/x/api/date-pickers/time-picker.json

```diff
@@ -84,6 +84,10 @@
     "orientation": {
       "type": { "name": "enum", "description": "'landscape'<br>&#124;&nbsp;'portrait'" }
     },
+    "reduceAnimations": {
+      "type": { "name": "bool" },
+      "default": "`@media(prefers-reduced-motion: reduce)` || typeof navigator !== 'undefined' && /(android)/i.test(navigator.userAgent)"
```

Comment:
I have tested https://deploy-preview-9858--material-ui-x.netlify.app/x/react-date-pickers/date-picker/#basic-usage with `<DatePicker reduceAnimations={false}` with my Android phone, the month switch transition feels in part with the native Android `<input type="date">`. So I think that we can remove the `/(android)/i.test(navigator.userAgent)` condition.

Reply:
Yes, I have also double-checked, that the native Android picker does also have the same calendar transition animations.


> I also don't understand why this transition could be slow, if we rely on the GPU to compose layers, it should be fine, no? https://www.smashingmagazine.com/2016/12/gpu-animation-doing-it-right/

On regular (decently powerful current) devices it should never be a problem, but when we consider devices from 5-6 years ago, especially cheap Android examples—that's another topic. 🙈 
I was considering changing the existing userAgent rule to a bit more elaborate/modern one with a separate PR.
Something along the lines of default to `reduceAnimations` when:
- Android version <10 (this would hopefully account for >80% of old-slow devices, that got left behind in Android version updates. And even higher-end devices like Galaxy S8 could struggle sometimes, just because it might have an old battery, which forces the device to throttle performance)
- iOS version <15 (iPhone 7 got the v15 update, whereas 6 got left behind)

What do you think? 🤔

Reply:
Sounds great 👍